src/id.rs

```diff
@@ -73,12 +76,20 @@ impl ID {
     }
 }
 
+impl From<ID> for Uuid {
+    #[inline]
+    fn from(id: ID) -> Self {
+        Uuid::from_u128(id.0.get())
+    }
+}
+
 impl From<Uuid> for ID {
     #[inline]
     fn from(uuid: Uuid) -> Self {
-        uuid.as_bytes()
+        uuid.as_u128()
+            .to_le_bytes()
```

Comment:
Uuid has a [to_u128_le()](https://docs.rs/uuid/latest/uuid/struct.Uuid.html#method.to_u128_le) operation. Would it be equivalent to this ?